CRM/Activity/BAO/Query.php

```diff
@@ -190,15 +190,14 @@ public static function whereClauseSingle(&$values, &$query) {
       case 'activity_type_id':
       case 'activity_status_id':
       case 'activity_engagement_level':
-      case 'activity_subject':
```

Comment:
Although there is a api test to assert ```subject``` but will it cause any unintended regression ?